src/Sylius/Behat/Page/Admin/Promotion/CreatePage.php

```diff
@@ -247,4 +272,43 @@ private function getCollectionItems($collection)
 
         return $items;
     }
+
+    /**
+     * @param NodeElement $autoComplete
+     * @param string $value
+     */
+    private function selectAutoCompleteValue(NodeElement $autoComplete, $value)
```

Comment:
Isn't it generic enough to extract it to a service? Or is it bound to promotions' create page?